packages/access-policy/src/__tests__/AccessPolicy.test.js

```diff
@@ -0,0 +1,128 @@
+    });
+  });
+
+  describe('allowsSome', () => {
+    describe('no permission group', () => {
+      const testData = [
+        [
+          'should allow access to an entity that has at least one permission group, when no permission group is specified',
+          ['DL'],
+          true,
+        ],
+        ['should not allow access to an entity that has no permission groups', ['VU'], false],
+        ['should not allow access with no entity', [], false],
+        ['should not allow access with null', null, false],
+        ["should not allow access with ''", '', false],
+      ];
+
+      it.each(testData)('%s', (_, entities, expected) => {
+        expect(accessPolicy.allowsSome(entities)).toBe(expected);
+      });
+    });
+
+    describe('with permission group', () => {
+      const testData = [
+        [
+          'should allow access to a permission group when a matching entity is specified',
+          [['DL'], 'Public'],
+          true,
+        ],
+        [
+          'should allow access to a permission group when multiple entities are specified',
+          [['DL', 'DL_North_West', 'VU'], 'Public'],
+          true,
+        ],
+        [
+          'should not allow access to a permission group that exists for a different entity than those specified',
+          [['DL', 'DL_North_West', 'VU'], 'Admin'],
+          false,
+        ],
+        [
+          'should allow access to a permission group that includes spaces',
+          [['SB'], 'Royal Australasian College of Surgeons'],
+          true,
+        ],
+        [
+          'should allow access if no entities are specified, along with a permission group',
+          [undefined, 'Royal Australasian College of Surgeons'],
+          true,
+        ],
+        [
+          'should allow access if no entities are specified, along with a permission group',
+          [null, 'Royal Australasian College of Surgeons'],
+          true,
+        ],
+      ];
+
+      it.each(testData)('%s', (_, [entities, permissionGroup], expected) => {
+        expect(accessPolicy.allowsSome(entities, permissionGroup)).toBe(expected);
+      });
+    });
+  });
+
+  describe('getPermissionGroups', () => {
+    it('should return all permission groups when no entities are specified', () => {
+      expect(accessPolicy.getPermissionGroups()).toStrictEqual(
+        expect.arrayContaining(['Public', 'Admin', 'Royal Australasian College of Surgeons']),
+      );
+    });
+
+    describe('should return just the permission groups that relate to the provided entities', () => {
+      const testData = [
+        ['entity (i)', ['KI'], ['Public', 'Admin']],
+        ['entity (ii)', ['DL', 'KI'], ['Public', 'Admin']],
+        [
+          'entity (iii)',
+          ['SB', 'KI'],
+          ['Public', 'Admin', 'Royal Australasian College of Surgeons'],
+        ],
+      ];
+
+      it.each(testData)('%s', (_, arr, expected) => {
```

Comment:
```suggestion
      it.each(testData)('%s', (_, entities, expected) => {
```

And in the line below